contracts/HATVault.sol

```diff
@@ -214,7 +214,7 @@ contract HATVault is IHATVault, ERC4626Upgradeable, OwnableUpgradeable, Reentran
             revert NotSafetyPeriod();
         if (_bountyPercentage > maxBounty)
             revert BountyPercentageHigherThanMaxBounty();
-        if (maxBounty == HUNDRED_PERCENT && _bountyPercentage != HUNDRED_PERCENT)
+        if (maxBounty == HUNDRED_PERCENT && _bountyPercentage != HUNDRED_PERCENT && _bountyPercentage > MAX_BOUNTY_LIMIT)
             revert PayoutMustBeHundredPercent();
```

Comment:
Maybe rename the error to `PayoutMustBeUpToMaxBountyLimitOrHundredPercent`
Though that's a bit long :)